src/winter_onboarding_2021/fleet_management_service/models/fleet.clj

```diff
@@ -2,10 +2,21 @@
   (:require [winter-onboarding-2021.fleet-management-service.db.fleet :as fleet-db]
             [winter-onboarding-2021.fleet-management-service.utils :as utils]
             [winter-onboarding-2021.fleet-management-service.specs :as specs]
-            [winter-onboarding-2021.fleet-management-service.error :as error]))
+            [winter-onboarding-2021.fleet-management-service.error :as error]
+            [winter-onboarding-2021.fleet-management-service.config :as config]))
 
 (defn create [fleet]
   (let [fleet-with-valid-keys (utils/select-keys-from-spec fleet ::specs/fleets)]
     (if (empty? fleet-with-valid-keys)
       error/no-valid-keys
       (fleet-db/create fleet-with-valid-keys))))
+
+(defn total []
+  (fleet-db/total))
```

Comment:
The `total` fn at the domain layer can and should probably unwrap the total fleet count from the database structures that we get it in.

Reply:
This makes sense since the domain layer should return only what is expected & is useful to its consumers.